libs-scala/gatling-utils/src/main/scala/com/daml/gatling/stats/CombinedStats.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.gatling
+package stats
+
+import scalaz.\/
+
+case class CombinedStats(
+    fixTrades: FixTradeStats,
```

Comment:
Let's remove all references to fix trades, as this is specific to the CHESS project.

Reply:
yep, it is gone. Thanks!